plugins/debuginfo-install.py

```diff
@@ -124,3 +127,5 @@ def _di_install(self, package):
                 continue
             self.dbgdone.append(dbgname)
             break
+        else:
+            logger.info(_("Could not find debuginfo for pkg: %s") % package)
```

Comment:
It's the message for user, I'd prefer not to use abbreviations:
Could not find debuginfo for package: %s